src/app/debug/debug-tree/debug-tree.component.ts

```diff
@@ -47,30 +61,22 @@ export class DebugTreeComponent implements OnDestroy {
     this.showMultipleAtATimeSubscription.unsubscribe();
   }
 
-  @Input() set currentView(value: any) {
-    if (this._currentView !== value) {
-      // TODO: Check if the current reports are part of the view
```

Comment:
Has this TODO been solved or an issue created? If so, please link the issue in this pull request.

Reply:
This has been fixed, I used TODOs so i could personally keep track of what i still had to do and this was one i forgot to get rid of it seems